app/javascript/components/Link/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import classNames from "classnames"
+import "./index.scss"
+
+export enum LinkSize {
+  ExtraSmall = "xs",
+  Small = "sm",
+  Base = "bs",
+}
+
+export enum LinkVariant {
+  Neutral = "neutral",
+  NeutralLight = "neutral-light",
+  NeutralDark = "neutral-dark",
+  Info = "info",
+}
+
+export interface LinkProps extends React.HTMLAttributes<HTMLAnchorElement> {
+  variant: LinkVariant
+  text: string
+  size: LinkSize
+  underline?: boolean
+}
+
+export default function Link(props: LinkProps): JSX.Element {
+  const { underline, size, className, variant, text, ...rest } = props
+
+  const usedClassName = classNames("Link", className, {
```

Comment:
I've added `className` option, in muffibook example it is not supported. This adds possibility to pass additional classnames to react_component

Reply:
Great idea :+1:, the components should be as flexible as possible.